src/components/questionary/Questionary.tsx

```diff
@@ -60,7 +60,7 @@ function Questionary({
   const classes = useStyles();
   const { state, dispatch } = useContext(QuestionaryContext);
   const isUserOfficer = useCheckAccess([UserRole.USER_OFFICER]);
-  const titleRef = useRef<HTMLElement | null>(null);
+  const titleRef = useRef<HTMLHeadingElement | null>(null);
```

Comment:
I had to make this change to allow me to add the `component` attribute to the Typography component this is used in.

I'm not sure if this has any wider implications? It didn't seem to in manual testing...